vochain/scrutinizer/process.go

```diff
@@ -358,6 +426,38 @@ func (s *Scrutinizer) newEmptyProcess(pid []byte) error {
 		RollingCensusSize: p.GetRollingCensusSize(),
 	}
 	log.Debugf("new indexer process %s", proc.String())
+
+	queries, ctx, cancel := s.timeoutQueries()
+	defer cancel()
+	if _, err := queries.CreateProcess(ctx, scrutinizerdb.CreateProcessParams{
+		ID:                pid,
```

Comment:
queries.CreateProcess could get `(ctx, p)` and build scrutinizerdb.CreateProcessParams{} itself. I think that would remove noise from process.go (where you don't care about database types).

Reply:
Same as above; CreateProcess is generated code. I don't dictate how it works or what its function type is :)